packages/pigeon/lib/kotlin_generator.dart

```diff
@@ -98,7 +98,7 @@ void _writeCodec(Indent indent, Api api, Root root) {
 
     indent.write(
         'override fun writeValue(stream: ByteArrayOutputStream, value: Any?) ');
-    indent.writeScoped('{', '}', () {
+    indent.scoped('{', '}', () {
```

Comment:
How is this change related to the PR?

Reply:
this change aim to format the code style like 
`override fun writeValue(stream: ByteArrayOutputStream, value: Any?) {`
but the generated code is a char: `\t`. i have drop commit and open another pr later.